### PR TITLE
New hearts

### PR DESCRIPTION
**Pull request by careeoki**
New icons, 10x10 and at the same pixel scale as the rest of the UI.
image
However, while reordering UI, the titlebar looks like this. I'm not smart enough to figure out the problem, unfortunately :-)
image

### CODE REVIEW

**Comment by IntegratedQuantum** [comment]
The titlebar looks like this because the window itself is smaller than the title bar height, but the border around the window still tries to display the actual dimension.
I think you could set the default scale to 0.5 and render everything in 20×20, that should make the titlebar 2× smaller without changing anything else.

Also it seems like you changed the line endings of the file again.

**Comment by careeoki** [comment]
That works, thanks.

File: src/gui/windows/healthbar.zig [hunk 1]
```diff
@@ -46,24 +46,24 @@ pub fn render() void {
 	var y: f32 = 0;
 	var x: f32 = 0;
 	var health: f32 = 0;
-	while(health < main.game.Player.maxHealth) : (health += 1) {
-		if(x >= window.contentSize[0]) {
+	while (health < main.game.Player.maxHealth) : (health += 1) {
```
Comment (IntegratedQuantum) [inline src/gui/windows/healthbar.zig:49]: you still seem to have lots of unintended whitespace changes, probably still from when you used the formatter.

Comment (careeoki) [reply src/gui/windows/healthbar.zig]: I did the best I can do. I have to go now

Comment (IntegratedQuantum) [reply src/gui/windows/healthbar.zig]: Looks good now, thanks.

**Pull request merged**